packages/ckeditor5-dev-transifex/lib/transifexservice.js

```diff
@@ -3,8 +3,8 @@
  * For licensing, see LICENSE.md.
  */
 
-const { transifexApi } = require( '@transifex/api' );
-const fetch = require( 'node-fetch' );
+import fetch from 'node-fetch';
```

Comment:
AFAIR, you dropped `node-fetch` in the past in another PR. We should be consistent and remove it from this package, too.